unlock-protocol.com/blog/hackfs.md

```diff
@@ -0,0 +1,22 @@
+---
+title: HackFS
+subTitle: Unlock is a sponsor for HackFS, and the Unlock community can vote on the best projects!
+authorName: Julien Genestoux
+publishDate: August 17, 2021
+description: At Unlock, we believe that hackers and makers will be building a better version of the web. For this reason, we are sponsors for HackFS, and we give our community the power to vote on the best projects!
+image: https://avatars2.githubusercontent.com/u/35270686?s=280&v=4
+---
+
+[HackFS](https://hackfs.com/) is a hackathon organized by the EthGlobal and Protocol Labs teams. The goal of the hackathon is to build the foundation for the decentralized web! At Unlock, we embrace this mission and have decided to join by sponsoring the hackathon!
+
+The hackathon started on July 30th and, teams have now submitted their projects! We're very excited to see that 17 teams have decided to build on top of the Unlock Protocol! You can view all of their submissions [on this Airtable](https://airtable.com/shrN4lIsh3yXQLQKh/tblfEgiZwI8Iog8ZK).
+
+It is very important for us to involve the Unlock community on deciding what are the best ways to promote and grow adoption! For this reason, a few weeks ago we announced that we would [decentralize our grants review process](https://unlock.community/t/decentralizing-the-grants-process/106)!
+
+Today, we want to try something new: let all of our community members vote on what the best projects from HackFS are!
+
+We have created a [Snapshot proposal](https://snapshot.org/#/unlock-community.eth) to let all existing community members vote! If you did not have a valid membership at the time of Snapshot, you won't be allowed to vote, but you should [get a membership](https://app.unlock-protocol.com/checkout?redirectUri=https%3A%2F%2Fsnapshot.org%2F%23%2Funlock-community.eth%2F&paywallConfig=%7B%22locks%22%3A%7B%220xac1fceC2e4064CCd83ac8C9B0c9B8d944AB0D246%22%3A%7B%22network%22%3A100%7D%7D%2C%22persistentCheckout%22%3Atrue%2C%22icon%22%3A%22https%3A%2F%2Flocksmith.unlock-protocol.com%2Flock%2F0xac1fceC2e4064CCd83ac8C9B0c9B8d944AB0D246%2Ficon%22%7D) for next time!
+
+If you are a member of our community, please, [vote now](https://snapshot.org/#/unlock-community.eth)!
+
+We will make sure that the most upvoted project gets a prize!
```

Comment:
Description: Our mission at Unlock is to empower hackers and makers to build a better version of the web. We chose to sponsor HackFS for this very reason and we wish to give our community the power to vote on the best projects.


EthGlobal and Protocol Labs created [HackFS](https://hackfs.com/), a hackathon designed to enable hackers and makers to lay the foundation for a decentralized web! Unlock embraces this mission completely and decided to help accelerate this movement by sponsoring the hackathon!

Hackathon teams began their work on July 30th and every team has submitted their projects for review by the judges. We're so excited to see that 17 teams chose Unlock Protocol to build their projects upon and all of their submissions are linked [on this Airtable](https://airtable.com/shrN4lIsh3yXQLQKh/tblfEgiZwI8Iog8ZK).

As a community-driven protocol, we want all of our community members to participate in deciding the best ways to promote and grow Unlock adoption. For this very reason, we recently decided to [decentralize our grants review process](https://unlock.community/t/decentralizing-the-grants-process/106).

Today we wish to try something new and further empower our community by letting members vote on the best projects from HackFS! The voting process is simple.

We have created a [Snapshot proposal](https://snapshot.org/#/unlock-community.eth) to let all community members vote! In order to vote, you need an Unlock NFT membership to this lock.

1. [Get a membership!](https://app.unlock-protocol.com/checkout?redirectUri=https%3A%2F%2Fsnapshot.org%2F%23%2Funlock-community.eth%2F&paywallConfig=%7B%22locks%22%3A%7B%220xac1fceC2e4064CCd83ac8C9B0c9B8d944AB0D246%22%3A%7B%22network%22%3A100%7D%7D%2C%22persistentCheckout%22%3Atrue%2C%22icon%22%3A%22https%3A%2F%2Flocksmith.unlock-protocol.com%2Flock%2F0xac1fceC2e4064CCd83ac8C9B0c9B8d944AB0D246%2Ficon%22%7D)

2. [Vote now](https://snapshot.org/#/unlock-community.eth)!

We will make sure that the most upvoted project gets a prize!